projects/discussion-ui/src/lib/components/my-discussion/my-discussion.component.ts

```diff
@@ -133,8 +133,8 @@ export class MyDiscussionComponent implements OnInit {
 
   navigateToDiscussionDetails(discussionData) {
     console.log('discussionData', discussionData);
-    let routerSlug = this.configService.getConfig().routerSlug ? this.configService.getConfig().routerSlug : ''
-    this.router.navigate([`${routerSlug}${CONSTANTS.ROUTES.TOPIC}${_.get(discussionData, 'topic.slug')}`]);
+    // let routerSlug = this.configService.getConfig().routerSlug ? this.configService.getConfig().routerSlug : ''
```

Comment:
do we need this comment? if not remove it if its required mention some comment TODO: